classes/label.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, color)
+    @id = rand(10_000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
As it is stated in the requirements `should add self as a property of the item object (by using the correct setter from the item object)` you should add self as a property in your add_item method.